shell/pages/support/index.vue

```diff
@@ -65,7 +64,6 @@ export default {
       supportSetting:       null,
       brandSetting:         null,
       uiIssuesSetting:      null,
-      communitySetting:     null,
```

Comment:
`return options(true, this.uiIssuesSetting?.value, this.communitySetting?.value === 'false');` also needs updating